### DISCUSSION TITLE
Zigbee Network Scan with Routes

### DISCUSSION THREAD

**Discussion by Hotte37** [Koenkk/zigbee2mqtt#6260] [General]
I use the network map for diagnosis purposes. For this i need to see the routes, but if i display the map i see the message "Starting network scan (includeRoutes 'false')". I was searching for a parameter in the config file for "includeRoutes 'true'", but i cannot find it.
Does the possibility exist to include also routes?

**Comment by oddyseus** [comment]
Bump up

**Comment by Koenkk** [comment]
It is possible to include routes via MQTT, but not sure if this can be done via the frontend. Please open an issue here: https://github.com/nurikk/z2m-frontend/issues

**Comment by RobSmyth** [comment]
The answer (not tested) might be to set a `routes` configuration option to `true`. See: https://github.com/Koenkk/zigbee2mqtt/discussions/18641.
